package.json

```diff
@@ -9,6 +9,7 @@
     "react": "^17.0.1",
     "react-dom": "^17.0.1",
     "react-scripts": "4.0.3",
+    "typescript": "^4.2.2",
```

Comment:
This seems to be working well enough, but I should look for where I'm getting all the esLint errors.